src/Uno.UWP/Devices/Bluetooth/GenericAttributeProfile/GattCharacteristicsResult.cs

```diff
@@ -8,5 +9,11 @@ public partial class GattCharacteristicsResult
 		public IReadOnlyList<GattCharacteristic> Characteristics { get; internal set; }
 		public  byte? ProtocolError { get; internal set; }
 		public  GattCommunicationStatus Status { get; internal set; }
+
+		private GattCharacteristicsResult(IReadOnlyList<GattCharacteristic> param)
+		{
+			// dummy for Error CS8618  Non-nullable property 'Value' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.
+			Characteristics = param;
```

Comment:
Let's create an empty list by default instead.

Reply:
This constructor probably would be used in 'real code'.
GattCharacteristicsResult cannot be created, it can only be returned inside other structs/classes from Bluetooth implementation.